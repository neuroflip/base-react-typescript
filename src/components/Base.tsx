import * as React from 'react'

import '../../css/Base.scss'

function Base (): React.JSX.Element {
  return <div>Hello from BASE!</div>
}

export default Base
