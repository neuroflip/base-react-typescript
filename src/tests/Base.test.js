/**
 * @jest-environment jsdom
 */

import * as React from 'react'
import renderer from 'react-test-renderer'
import Base from '../components/Base'

let container = null

beforeEach(() => {
  container = window.document.createElement('div')
  container.id = 'root'
  window.document.body.appendChild(container)
})

afterEach(() => {
  window.document.body.removeChild(container)
})

it('Flatline Integration test', () => {
  const component = renderer.create(<Base></Base>)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
