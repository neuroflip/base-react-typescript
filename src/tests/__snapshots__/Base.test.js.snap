// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flatline Integration test 1`] = `
<div>
  Hello from BASE!
</div>
`;
